### DISCUSSION TITLE
<pnp-icon> with text

### DISCUSSION THREAD

**Discussion by KarolFilipKozlowski** [microsoft-search/pnp-modern-search#3894] [General]
Hi,
How can I set a field with [<pnp-icon>](https://microsoft-search.github.io/pnp-modern-search/extensibility/web_components_list/#pnp-icon) any text in one line?

Here is my handlebars code:
```
{{#if VMCPDTitleEnvironmentTyp}}
{{VMCPDTitleEnvironmentTyp}}
{{else}}
<span>A<pnp-icon data-name="TagUnknown" aria-hidden="true">C</pnp-icon>B</span>
{{/if}}
```

And how I see it on the results webpart:
![image](https://github.com/user-attachments/assets/9f9ccfbd-394e-45e6-b77f-355ebe5ebd1a)

I want to have A, B or C in the same line, without new line.

**Comment by kasperbolarsen** [comment]
There should be a lot of inspiration available at https://github.com/microsoft-search/pnp-modern-search-layouts
